src/nupic/regions/ScalarSensor.hpp

```diff
@@ -70,21 +70,11 @@ class ScalarSensor : public RegionImpl {
   getNodeOutputElementCount(const std::string &outputName) const override;
 
 private:
-  struct {
-    Real64 sensedValue_;
-    Real64 resolution;
-    Real64 radius;
-    Real64 minValue;
-    Real64 maxValue;
-    UInt32 n;
-    UInt32 w;
-    bool periodic;
-    bool clipInput;
-  } params_;
+  Real64 sensedValue_;
```

Comment:
is the last sensed value needed to be kept and serialized here?

Reply:
I think so. And the unit tests agree that the sensedValue should be saved, since they save/load a network and then run the net without assigning a new value.